<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projet de recherche d'informations</string>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="mainTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="searchTab">
       <attribute name="title">
        <string>Recherche</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="modelGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Modèle</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="tabPosition">
              <enum>QTabWidget::North</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="vectorTab">
              <attribute name="title">
               <string>Vectoriel</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="vectorSearchLayout">
                 <item>
                  <widget class="QLineEdit" name="vectorSearchLineEdit">
                   <property name="placeholderText">
                    <string>Votre requête</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="vectorSearchPushButton">
                   <property name="text">
                    <string>Rechercher</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QGroupBox" name="vectorSimilarityGroupBox">
                 <property name="title">
                  <string>Fonction de similarité</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QRadioButton" name="innerProductRadioButton">
                    <property name="text">
                     <string>Produit interne</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="diceRadioButton">
                    <property name="text">
                     <string>Dice</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="cosRadioButton">
                    <property name="text">
                     <string>Cosinus</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="jaccardRadioButton">
                    <property name="text">
                     <string>Jaccard</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="booleanTab">
              <attribute name="title">
               <string>Booléen</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="booleanSearchLayout">
                 <item>
                  <widget class="QLineEdit" name="booleanSearchLineEdit">
                   <property name="placeholderText">
                    <string>Votre requête</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="booleanSearchPushButton">
                   <property name="text">
                    <string>Rechercher</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="booleanNoteLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Les opérateurs booléen : &amp;amp; (et), | (ou), ~ (non).&lt;/p&gt;&lt;p&gt;Les parenthèses peuvent être utilisées pour indiquer la priorité.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="matchingScoreTab">
              <attribute name="title">
               <string>Matching score</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="matchingScoreSearchLayout">
                 <item>
                  <widget class="QLineEdit" name="matchingScoreSearchLineEdit">
                   <property name="placeholderText">
                    <string>Votre requête</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="matchingScoreSearchPushButton">
                   <property name="text">
                    <string>Rechercher</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ResultsGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Résultats</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableWidget" name="resultsTableWidget">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <column>
              <property name="text">
               <string>Document</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Score</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearResultsPushButton">
             <property name="text">
              <string>Effacer</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="extraTab">
       <attribute name="title">
        <string>Fichier inverse</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="loadInverseFileGroupBox">
          <property name="title">
           <string>Charger un fichier inverse</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="loadInverseFileLineEdit">
             <property name="placeholderText">
              <string>Chemin du fichier</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loadInveseFilePushButton">
             <property name="text">
              <string>Sélectionner</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="saveInverseFileGroupBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Sauvegarder un fichier inverse</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="saveInverseFileFieldLayout">
             <item>
              <widget class="QLineEdit" name="saveInverseFileLineEdit">
               <property name="placeholderText">
                <string>Chemin du fichier</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="saveInveseFilePushButton">
               <property name="text">
                <string>Sélectionner</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="saveInverseFileGeneratePushButton">
               <property name="text">
                <string>Générer</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="saveInvefrseFileChoiceLayout">
             <item>
              <widget class="QRadioButton" name="saveInverseFileTfIdfRadioButton">
               <property name="text">
                <string>Tf-Idf</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="saveInverseFileFrequencyRadioButton">
               <property name="text">
                <string>Fréquence</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Paramètres</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
